content/blog/projects/current/f1-10.md

```diff
@@ -15,14 +15,15 @@ head:
 ---
 
 # F1TENTH
-
+<p style="text-align:justify">
```

Comment:
As every blank-spaced block of text is already a paragraph, instead of putting whole text in another paragraph, request adding the following in `integra.front` stylesheet:

<img width="152" alt="image" src="https://github.com/KN-Integra/integra.content/assets/61759168/b325f291-a296-41f0-92b9-1a777774a6da">

Also request that `<a>` tags should be `inline` instead of `inline-block` for content pages.